lib/libzpool/uzfs_rebuilding.c

```diff
@@ -393,3 +393,49 @@ uzfs_zvol_get_or_create_internal_clone(zvol_state_t *zv,
 	strfree(snapname);
 	return (ret);
 }
+
+/*
+ * To destroy all internal created snapshot
+ * on a dataset
+ */
+int
+uzfs_destroy_internal_all_snap(zvol_state_t *zv)
```

Comment:
name change to 'uzfs_destroy_all_internal_snapshots'?

Reply:
ok